migrations/20170302131148_added_cities_table.js

```diff
@@ -0,0 +1,25 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('cities', table => {
+    table.increments('id').primary().index();
+    table.string('name').notNullable().index();
+  }).then(() => {
+    return knex.schema.table('teams', (table) => {
```

Comment:
Not sure if there's a lint rule for this in this project but my personal opinion is that single parameters in arrow functions shouldn't be in parenthesis :)

At least the first arrow function on line 3 doesn't have it